sklearn/utils/_testing.py

```diff
@@ -760,6 +776,10 @@ def assert_docstring_consistency(
         List of returns to be excluded. If None, no returns are excluded.
         Can only be set if `include_returns` is True.
 
+    description_regex : str, default=""
+        Regular expression to match to all descriptions. If empty string, will
+        revert to comparing descriptions between objects.
```

Comment:
I'm wondering if having `default=None` would be a better semantic instead of `""`. It would indicate that we don't use any pattern.